WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -69,4 +77,12 @@ extension ShippingLabelFormViewModel {
                                                         postcode: address.postcode)
         return shippingLabelAddress
     }
+
+    static func getStoredAccountSettings() -> AccountSettings? {
+        let storageManager = ServiceLocator.storageManager
+
+        let resultsController = ResultsController<StorageAccountSettings>(storageManager: storageManager, sortedBy: [])
+        try? resultsController.performFetch()
+        return resultsController.fetchedObjects.first
```

Comment:
non-blocking thought 💬 I wish we don't have to couple the view model with the storage layer here - the alternative is to retrieve the account settings via Yosemite layer like `AccountStore`, the downside is the result comes back asynchronously 😐

Reply:
I've thought about it, but I think this brings extra complexity in handling the things that maybe we can avoid right now.  This flow will be full of API requests, and having the information asynchronously will lead to many problems in verifying the address in the future, it is practically necessary to create a queue of calls, to make sure that the source address that is being verified is complete (and that therefore the information has been taken from the API). I would say not to discard this option, and to evaluate it as soon as we have a complete view of the flow for verifying the address, so as to understand if it is easily integrated, or not necessary. 👍